scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/mtls/K8sCertSecretsProvider.kt

```diff
@@ -31,7 +31,7 @@ import java.nio.file.Files
 object K8sCertSecretsProvider {
 
     private val kubeConfigPath: String = System.getenv("HOME") + "/.kube/config"
-    private val namespace = System.getenv("POD_NAMESPACE")
+    private val namespace = System.getenv("SELDON_POD_NAMESPACE")
```

Comment:
:thought_balloon: I'll put in a small follow-up PR to move these to use the CLI-provided value